examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,291 @@
+"""
+Multiphysics: Maxwell 3D - Icepak electrothermal analysis
+---------------------------------------------------------
+This example uses PyAEDT to set up a simple Maxwell design consisting of a coil and a ferrite core.
+Coil current is set to 100A, and coil resistance and ohmic loss are analyzed.
+Ohmic loss is mapped to Icepak, and a thermal analysis is performed. Icepak calculates a temperature distribution,
+and it is mapped back to Maxwell (2-way coupling). Coil resistance and ohmic loss are analyzed again in Maxwell.
+Results are printed in AEDT Message Manager.
+Keywords: Litz wire model, Icepak coupling, 2-way coupling
+"""
+###########################################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+
+import pyaedt
+from pyaedt.generic.constants import AXIS
+
+###########################################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode.
+# You can set ``non_graphical`` either to ``True`` or ``False``.
+
+non_graphical = False
+
+###########################################################################################
+# Launch AEDT and Maxwell 3D
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Launch AEDT and Maxwell 3D. The following code sets up the project and design names, the solver, and
```

Comment:
Recommend writing the description in the same order as the actions performed by the code.